interfaces/zookeeper_client/v0/interface_tests/README.md

```diff
@@ -0,0 +1,33 @@
+This directory is meant to contain the tests for your interface, so that it is possible to programmatically verify its implementations.
+
+Find some relevant documentation at:
+- [Interface Tests](https://discourse.charmhub.io/t/interface-tests/12691)
+- [How to write interface tests](https://discourse.charmhub.io/t/how-to-write-interface-tests/12690)
+
+## Quickstart
+
+Copy this test to `test_provider.py` and fill in the blanks however appropriate for your interfaces
+
+```python
+from scenario import Relation, State
+from interface_tester import Tester
+
+def test_data_published_on_changed_remote_valid():
+    # GIVEN a relation over <interface> containing all the right data 
+    #  in the right format for the <requirer side>:
+    relation = Relation(
+        endpoint='ingress',
```

Comment:
`ingress`? Is this a mistake?

Reply:
This comes from the template for new interfaces, tests are not implemented yet

Reply:
Testing the interface turned out to be quite difficult because our charm are designed in a way that the relation databag is populated over multiple invocations. According to Pietro, the interface tester was not created with this complex setup in mind. 
I removed the tests from this repository.